device/arista/x86_64-arista_7800r3_48cq2_lc/Arista-7800R3-48CQ2-C48/buffers.json.j2

```diff
@@ -0,0 +1,2 @@
+{%- set default_topo = 't0' %}
```

Comment:
't2'?

Reply:
@smaheshm Thanks for catching this. Will fix.

Reply:
@ysmanman , Please let us know once done. Planning to start verification for these settings.

Reply:
@vmittal-msft Updated the review to address the above comment. We were testing PFC with a SAI patch from Broadcom ( Broardcom case: CS00012207728).  You may need this patch as well if you want test PFC. Add @rlhui for viz.

Reply:
@ysmanman Thanks for quick response. I am unable to access CS00012207728. Is it possible for you to share some detail ?

Thanks
Vineet

Reply:
@vmittal-msft We uncovered some issues in PFC testing, e.g., switch didn't send pause frame to to upstream even it received pause frames from downstream in certain scenarios; and storm mitigation was not working (switch kepts sending pause frame even it's in stormed state) when PFCWD was enabled. We filed few Broadcom cases, CS00012203593 and CS00012207728. Broadcom provides a patch (on top of SAI 5.2) for above issues, and we have been testing the patch recently.